package.json

```
@@ -63,6 +62,7 @@
     "picocolors": "^1.0.1",
     "taze": "^0.16.3",
     "terminal-link": "^3.0.0",
+    "tinyexec": "^0.1.4",
```

new version is out now that fixes the timeout
```suggestion
    "tinyexec": "^0.2.0",
```